src/luxonis_ml/scheme/v1/config_building_blocks/enums/decoding.py

```diff
@@ -0,0 +1,30 @@
+from enum import Enum
+
+class DecodingFamily(Enum):
+
+    """ members of a family should have similar decoding """
+
+    # classification
+    MULTICLASS_CLASSIFICATION= "multiclass_classification"
+
+    # object detection
+    YOLO = "yolo"
+    SSD_PARSED = "ssd-mobilenet-parsed" # seems like outputs of all ssd-mobilenet networks are of shape (1,3000,classes_n) and (1,3000,4)
+    
+    # keypoints
+    KEYPOINTS_HEATMAP = "heatmap"
+    KEYPOINTS_REGRESSION = "regression"
+    KEYPOINTS_ANCHORS = "anchors"
+    
+    # segmentation
+    # TODO
+
+    # misc
+    RAW = "raw"
+
+class ObjectDetectionSubtype(Enum):
+
+    """ subtype members have exactly the same decoding """
+
+    YOLOv5_7 = "yolov5/yolov7"
```

Comment:
Let's have those separate. So YOLOv5, and same for V6, V7, and V8. Even if decoding is similar, I think it's good to differentiate still.